gen/dcos-config.yaml

```diff
@@ -1497,6 +1497,7 @@ package:
         "diagnostics-bundle-dir": "{{ diagnostics_bundles_dir }}",
         "endpoint-config": "/opt/mesosphere/etc/dcos-diagnostics-endpoint-config.json",
         "no-unix-socket": true,
+        "fetchers-count": 100,
```

Comment:
This number is arbitrary chosen. We have a time to tune it before 1.14 release.

Reply:
Let's please have a ticket for tracking tuning of this parameter in our next sprint then.

Reply:
https://jira.mesosphere.com/browse/DCOS_OSS-5077